core/tests/protractor_desktop/profileFeatures.js

```diff
@@ -15,3 +15,105 @@
 /**
 * @fileoverview End-to-end tests for user profile features.
 */
+
+var general = require('../protractor_utils/general.js');
+var users = require('../protractor_utils/users.js');
+var workflow = require('../protractor_utils/workflow.js');
+
+
+var LearnerDashboardPage =
+  require('../protractor_utils/LearnerDashboardPage.js');
+var LibraryPage = require('../protractor_utils/LibraryPage.js');
+var SubscriptionDashboardPage =
+  require('../protractor_utils/SubscriptionDashboardPage.js');
+
+describe('Profile Features functionality', function() {
+  var libraryPage = null;
+  var learnerDashboardPage = null;
+  var subscriptionDashboardPage = null;
+
+  var creator1Id = 'FirstCreator';
+  var creator2Id = 'SecondCreator';
+  users.createUser('FirstUser@profileFeatures.com',
+    'FirstUserprofileFeatures');
+  users.createUser('SecondUser@profileFeatures.com',
+    'SecondUserprofileFeatures');
+  users.createUser(creator1Id + '@profileFeatures.com',
+    creator1Id);
+  users.createUser(creator2Id + '@profileFeatures.com',
+    creator2Id);
+
+  beforeAll(function() {
+    libraryPage = new LibraryPage.LibraryPage();
+    learnerDashboardPage = new LearnerDashboardPage.LearnerDashboardPage();
+    subscriptionDashboardPage =
+      new SubscriptionDashboardPage.SubscriptionDashboardPage();
+  });
+
+  it('View a particular exploration created by another User', function() {
+    users.login(creator1Id + '@profileFeatures.com');
+    workflow.createAndPublishExploration(
+      'Activation',
+      'Chemistry',
+      'English'
+    );
+    users.logout();
+
+    users.login('FirstUser@profileFeatures.com');
+    // Subscribing to a User to get all his created explorations.
+    subscriptionDashboardPage.navigateToUserSubscriptionPage(creator1Id);
+    subscriptionDashboardPage.navigateToSubscriptionButton();
```

Comment:
This function doesn't seem to be named correctly (I think this is a problem in the existing code). I don't think you can "navigate" to a button. Maybe the right naming is clickSubscriptionButton or something like that :)

Reply:
I've checked the function definition before itself, it includes `subscriptionButton.click()`

Reply:
Yeah, I noticed the click statement too. I was referring to the name itself. I am happy to leave it as is for now.